app/connectors/CustomsDeclarationsConnector.scala

```diff
@@ -31,23 +31,25 @@ import scala.xml.XML
 
 class CustomsDeclarationsConnector @Inject()(config: AppConfig, httpClient: HttpClient)(implicit ec: ExecutionContext) {
 
+  private val logger = Logger(this.getClass)
+
   private val fileUploadUrl = config.microservice.services.customsDeclarations.batchUploadEndpoint
   private val apiVersion = config.microservice.services.customsDeclarations.apiVersion
   private val clientId = config.developerHubClientId
 
   def requestFileUpload(eori: String, request: FileUploadRequest)(implicit hc: HeaderCarrier): Future[FileUploadResponse] = {
-    Logger.info(s"Request to initiate ${request.toXml}")
-    Logger.info(s"fileUploadUrl: $fileUploadUrl")
+    logger.debug(s"Request to initiate ${request.toXml}")
```

Comment:
Don't change info to debug as once in ET/prod debug will not be available in PROD